contracts/proposals/lip004.clar

```diff
@@ -0,0 +1,21 @@
+
+;; SPDX-License-Identifier: BUSL-1.1
+
+(impl-trait 'SM26NBC8SFHNW4P1Y4DFH27974P56WN86C92HPEHH.proposal-trait.proposal-trait)
+
+(define-public (execute (sender principal))
+	(begin
+		(try! (contract-call? 'SM26NBC8SFHNW4P1Y4DFH27974P56WN86C92HPEHH.lisa-dao set-extensions (list
```

Comment:
It would be nice to have a view of the current enabled extensions somewhere

Reply:
Yes it's something we can make.